src/components/connect.js

```diff
@@ -208,13 +173,23 @@ export default function connect(mapStateToProps, mapDispatchToProps, mergeProps,
         }
       }
 
+      componentWillMount() {
```

Comment:
@gaearon I've been able to remove the store subscription here, but I still need to be able to trigger the computation from here, otherwise nothing can be put into state for the first render and we are back to doing the computation in render()

is that ok?